h/schemas/admin_group.py

```diff
@@ -17,18 +17,39 @@
 _ = i18n.TranslationString
 
 VALID_GROUP_TYPES = (
-    ('private', _('Private')),
     ('restricted', _('Restricted')),
     ('open', _('Open')),
 )
 
 
+def _split_origins(value):
+    if value != colander.null:
+        return [origin.strip() for origin in value.splitlines()]
+    return value
+
+
+def user_exists_validator_factory(user_svc):
```

Comment:
I designed this this way such that the schema didn't have a dependency on any services/request-related things.

Reply:
This looks good to me. On the call yesterday I suggested moving the semantic validation up to the view level - but this has an important disadvantage that I completely forgot about: Doing validation of whether usernames, authorities exist etc. at the field level in the Colander schema results in error messages being displayed in a helpful way after submitting the form.

There is actually plenty of precedence for doing this kind of validation in the schema, but it is in `h.accounts.schemas` which I think predates the introduction of the `h.schemas` package.

So in short, I think we can keep this as-is.